templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/common/impl/config/ApplicationObjectMapperFactory.java

```diff
@@ -29,5 +32,17 @@ public ApplicationObjectMapperFactory() {
 	// register spring-data Pageable
     module.addSerializer(Pageable.class, new PageableJsonSerializer());
     module.addDeserializer(Pageable.class, new PageableJsonDeserializer());
+
+    ObjectMapper objectMapper = super.createInstance();
```

Comment:
@vapadwal almost perfect now, but this block needs to go to the method `createInstance` that is supposed to be overridden with this logic.